### PR TITLE
Task/Add translations function to /personal pages

### PR DESCRIPTION
**Pull request by pcraig3**
## Add translations functions to all the content for the /personal pages.

This includes:

- /personal/name
- /personal/residence
- /personal/address
- /personal/address/edit
- /personal/maritalStatus
- /personal/maritalStatus/edit

## Added new mixin for the link buttons that we have on several pages

Again, these are mostly copy and paste so the mixin makes sense

## Didn't translate the marital status radio buttons or the provinces select element

It's possible we'll have a better way of handling these in the near future (ie, #23). If not, we can get these manually translated next week, either Monday or Tuesday.

### CODE REVIEW

File: views/personal/address.pug [hunk 1]
```diff
@@ -26,13 +26,6 @@ block content
               div #{data.personal.address.city}, #{data.personal.address.province}
               div #{data.personal.address.postalCode}
 
-  div
-    details.
-      <summary><span>Help with mailing address</span></summary>
```
Comment (sboisvert) [inline views/personal/address.pug]: Why are we removing this?

File: views/personal/maritalStatus.pug [hunk 2]
```diff
@@ -5,27 +5,20 @@ block variables
...
 
-  div
-    details.
-      <summary><span>Help with marital status</span></summary>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-
-  .buttons-row
-    a.button-link(href='/review') Confirm
+  +linkButtons('/review')
     a.button-link(href='/personal/maritalStatus/edit') Change Marital Status
```
Comment (sboisvert) [inline views/personal/maritalStatus.pug]:
```suggestion
    a.button-link(href='/personal/maritalStatus/edit') #{__('Change Marital Status')}
```

File: views/personal/name.pug [hunk 3]
```diff
@@ -5,31 +5,28 @@ block variables
...
-      <summary><span>Help with name confirmation</span></summary>
-      <p>If you have changed your name since the last time you filed your taxes, <a href="https://www.canada.ca/en/revenue-agency/services/tax/individuals/frequently-asked-questions-individuals/should-you-tell-cra-about-your-change-name.html" target="_blank">get in contact with CRA to update your information</a>.</p>
-      <p>Once updated, you can proceed to use this service.</p>
-
-  .buttons-row
-    a.button-link(href='/personal/residence') Confirm
-    a.button-link(href='/offramp') My name has changed
-    a.button-link.transparent(href='/clear') Cancel
+      <summary><span>#{__('Help with name confirmation')}</span></summary>
+      <p>#{__('If you have changed your name since the last time you filed your taxes,')} <a href="https://www.canada.ca/en/revenue-agency/services/tax/individuals/frequently-asked-questions-individuals/should-you-tell-cra-about-your-change-name.html" target="_blank">#{__('get in contact with the CRA to update your information')}</a>.</p>
```
Comment (sboisvert) [inline views/personal/name.pug]: The url should probably be a translation

File: views/personal/residence.pug [hunk 4]
```diff
@@ -6,24 +6,15 @@ block variables
...
-  h1 #{title}
+  h1 #{__(title)}
 
   div
-    p In order to be eligible for this service, your <strong>primary residence</strong> on December 31, 2018, must have been in Ontario.
+    p #{__('In order to be eligible for this service, your primary residence on December 31, 2018, must have been in Ontario.')}
 
 
   form.cra-form(method='post')
     +yesNoRadios('residence', residence, 'Ontario resident on December 31, 2018?', errors)
```
Comment (sboisvert) [inline views/personal/residence.pug]: Is the translation module going to be able to figure out these strings that are passed as arguments or do we need to add those to the locale file manually?

File: views/personal/address.pug (hunk 1 above)
Comment (pcraig3) [reply views/personal/address.pug]: I asked @brianhendrick1 about it and he said that we don't have content for most of those, so to remove them.

File: views/personal/maritalStatus.pug (hunk 2 above)
Comment (pcraig3) [reply views/personal/maritalStatus.pug]: Good catch

File: views/personal/residence.pug (hunk 4 above)
Comment (pcraig3) [reply views/personal/residence.pug]: Yup, it's in there: https://github.com/cds-snc/cra-claim-tax-benefits/pull/49/files#diff-1d62a5823ca4461fb794fa0aa5de49eeR86

We do the same thing with our error messages: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/errorList.pug#L6

**Comment by pcraig3** [comment]
Okay @sboisvert, updated with your suggestions.

**Pull request merged**